examples/full/cmd/server.go

```diff
@@ -25,7 +26,7 @@ type Server struct {
 	TemplateFS fs.FS
 }
 
-func (s *Server) Run(ctxRoot context.Context) error {
+func (s *Server) Run(ctx context.Context) error {
```

Comment:
Does it make sense to pass ctx if you overwrite it 4 lines down?

Reply:
The overwrite takes this context as an argument.

Reply:
I forgot to delete some lines. It is fixed now. Thank you.